src/core/contracts/registry.ts

```diff
@@ -145,8 +145,8 @@ export const getDSNPRegistryUpdateEvents = async (
 };
 
 /**
- * isMessageSignatureAuthorizedTo() validates a serialized message or DSNPMessage against a signature and then checks that the
- * signer has the permissions specified.  DSNPMessages should be passed as is,
+ * isSignatureAuthorizedTo() validates a serialized message or announcement against a signature and then checks that the
```

Comment:
This new method name seems awfully similar to `isAuthorizedTo()`. I wonder if we should combine the two or maybe just scrap this method given that `recoverPublicKey()` is a method in `src/core/announcements/crypto.ts` and we could just pass the result of it to `isAuthorizedTo()`.

Reply:
In any case, I think we could save it for a subsequent PR, not this one.

Reply:
I like that split idea. I'll see about it in another rename pr